.travis.yml

```diff
@@ -9,10 +9,13 @@ before_install:
   - nvm use 8
   - npm install -g yarn
 
-script:
-  - make lint-frontend
-  - make test-frontend
-  - make dependencies-backend
-  # uncommend next line when BE code will start passing go lint and go vet
-  # - make lint-backend
-  - make test
+jobs:
+  include:
+    - stage: Tests, linters and build
+      script:
+        - make lint-frontend
+        - make test-frontend
+        - make lint-backend
+        - make test
+    - script:
+        - make build-app
```

Comment:
no need to create tars for testing.

Reply:
The issue requires to test the whole building process

Reply:
creating tars isn't building.

Reply:
Just called `make packages` and whatever it does, is ok for the purpose of the issue.
Is there any problem if the tars are generated?

Reply:
nothing really wrong. just my perfectionism 😅 
Task has the name `test build`. Comment in travis says `build`. And we have a target `make build` exactly for this purpose - to build.

Reply:
Where is the name `test build`?

aside:
`build` rule is called from `packages` rule, that is called from `build-app` that is what drone really calls, that is what we're trying to reproduce.

Reply:
I meant the name of the job:`stage: Tests, linters and build`.

As you may notice I didn't request changes, it's okay to merge it as it is.